src/styles/themes/touch.scss

```diff
@@ -0,0 +1,26 @@
+.theme-touch {
+    --body-background: #222;
+    --nav-background: #333;
+    --nav-background-rgb: 51,51,51;
+    --danger-background: #fa023c;
+    --danger-text-color: #fff;
+    --game-card-background: #2B2B2B;
+    --game-card-text-color: #fff;
+    --header-text-color: #fff;
+    --accent-color: #fc0;
+    --list-background: transparent;
+    --list-header-background: transparent;
+    --list-header-text-color: #fff;
+    --modal-background: #222;
+    --modal-text-color: #fff;
+    --note-color: #ffae42;
+    --primary-background: #333;
+    --primary-text-color: #c0c0c0;
+    --rating-star-color: #ffae42;
+    --secondary-background: #333;
+    --secondary-text-color: #fff;
+    --success-background: #0ab9e6;
+    --success-text-color: #414548;
+    --warning-background: #ffae42;
+    --warning-text-color: #734100;
+}
```

Comment:
I really like this color combination, I think you should definitely keep the theme.

However, regarding the UI changes based on the theme... I'm not a big fan of creating a different experience based on the theme selected. The more theme-related variations we introduce the more difficult will be to maintain the code base down the road.

Instead, I suggest you keep the theme (awesome colors!) and remove any UI changes, in exchange, I feel like those mobile enhancements should become new features/settings (e.g. header/dock position `top/bottom`) that could be applied to any theme. I just want to keep the experience the same regardless of the theme chosen. I hope that makes sense.

Reply:
The difference in colors between this one and the dark theme are very minimal. Would that still apply for a new theme in itself? If we remove the UI changes it's just the dark theme with more transparency at that point.
![Screenshot 2019-12-15 at 12 59 56 PM](https://user-images.githubusercontent.com/984069/70862299-e217db80-1f3a-11ea-8225-1ca90c1a91e4.png)

> I'm not a big fan of creating a different experience based on the theme selected. The more theme-related variations we introduce the more difficult will be to maintain the code base down the road.
>  I feel like those mobile enhancements should become new features/settings (e.g. header/dock position top/bottom) that could be applied to any theme. I just want to keep the experience the same regardless of the theme chosen.

That's fair. What's your thoughts about turning some of these into default mobile adjustments without any settings?

- keep the big "add new list" button (fourth mobile screenshot)
- move the "add game to list" button to the top next to the list settings button
- turn the gear into ellipsis (not shown in the screenshots, but I added that with 3584599)
- hide IGDB icon to save on "screen real estate"
- make lists as wide as the phone's screen
- bigger list names
- turn icons bigger

Those would only affect mobile. New settings would be:

- "app" mode
  - apply mobile adjustments to the desktop view
- header position
  - top (default)
  - bottom
- border-radius
  - on (default)
  - off

I would like to make the centered drag icons a default everywhere though. If that's a no-go, I'll add them to the mobile adjustments.

Reply:
Dark theme:
![Screenshot 2019-12-15 at 1 26 28 PM](https://user-images.githubusercontent.com/984069/70862592-a67f1080-1f3e-11ea-89a5-a2e6c4e3ddc2.png)

Touch theme:
![Screenshot 2019-12-15 at 1 26 42 PM](https://user-images.githubusercontent.com/984069/70862607-c1ea1b80-1f3e-11ea-988e-1dab02285317.png)

I feel like this could just turn into a setting to add transparency instead of its own theme. The differences are too minimal.

Reply:
I noticed that in the platform settings it says "Global theme". Wouldn't that be "Platform theme"?

Are there plans to apply the theme globally in the settings modal where the user can select the platforms?

Reply:
> I noticed that in the platform settings it says "Global theme". Wouldn't that be "Platform theme"?
That's leftover from my original idea of having a global theme and a platform theme, I dropped that idea because I wanted the themes to be focused on the game board and not so much in the platforms page.

> Are there plans to apply the theme globally in the settings modal where the user can select the platforms?
That's what the original idea was, I opted for keeping it simple and have the platforms page default to the `default` theme.

---

- keep the big "add new list" button (fourth mobile screenshot)
While I think it makes it easier to tap/click because of the greater area, I like the consistency of having all the buttons look and feel the same way.

- move the "add game to list" button to the top next to the list settings button
I used to have it up there at some point, but I noticed that it was truncating some list names, especially in other languages other than english (reducing real estate), I'm open to try it, but I prefer those changes to take place in a separate PR just so we can test all use cases and make sure it works great.

- turn the gear into ellipsis (not shown in the screenshots, but I added that with 3584599)
Same, I used to have it as an ellipsis before, I'm not opposed to go back to it, although I did notice that a lot of users pretty much ignored the ellipsis, the setting cog seems to have more user engagement for some reason.

- hide IGDB icon to save on "screen real estate"
Unfortunately, this is a requirement for being an IGDB partner, we can tweak its appearance but they do want me to display that credit as much as possible, and I'm happy to do so, after all this tool wouldn't exist without IGDB.

- make lists as wide as the phone's screen
I used to have it this way when game board had only 1 list, I ran into issues with the masonry view and instead of fine tuning it I decided to revert the change until I have the time to make it 100% perfect. I would say go for it, again it would be nice to have a separate smaller PR for that change.

- bigger list names
This one can be very subjective, I've played with the idea of allowing font customization but it would make it very hard to develop as you would have to test on every theme. Maybe we can increase it on mobile only?

- turn icons bigger
Open to it as long as it doesn't cause UI issues, I was thinking about maybe making the whole list header clickable instead of having an ellipsis/cog icon.

Reply:
> make lists as wide as the phone's screen

Actually, this is already in place, except that it only applies when list view is not masonry (`unique` class is applied)

<img width="612" alt="Screen Shot 2019-12-16 at 9 19 16 AM" src="https://user-images.githubusercontent.com/645310/70923492-2fe91e00-1fe5-11ea-8f6a-dba877a91fe0.png">

Reply:
>  >  keep the big "add new list" button (fourth mobile screenshot)
> 
> While I think it makes it easier to tap/click because of the greater area, I like the consistency of having all the buttons look and feel the same way.

It just looks a bit off on mobile with a small icon dangling there, especially now with the scroll snapping (but before, too). How about making the parent element of the single "add new list" button `100vw` on mobile? That way it would feel like a new area the user is swiping to that they can start working with.

Reply:
> > move the "add game to list" button to the top next to the list settings button
> 
> I used to have it up there at some point, but I noticed that it was truncating some list names, especially in other languages other than english (reducing real estate), I'm open to try it, but I prefer those changes to take place in a separate PR just so we can test all use cases and make sure it works great.

My idea for the "add game to list" button was that it should be in the same place for every list. If the user swipes from one list to another and the + button keeps jumping, that's a bit confusing. As a compromise, how about making the lists fill the whole height on mobile? That way the icon would always be on the bottom right.

Reply:
> > turn the gear into ellipsis (not shown in the screenshots, but I added that with [3584599](https://github.com/romancm/gamebrary/commit/35845999e672c1726278360f9fd13686e659a418))
> 
> Same, I used to have it as an ellipsis before, I'm not opposed to go back to it, although I did notice that a lot of users pretty much ignored the ellipsis, the setting cog seems to have more user engagement for some reason.

That's weird, but if the user engagement is up, then it must have a reason. The cog just looks like a global settings icon to me, not a list settings icon. Maybe we could try a [sliders](https://fontawesome.com/icons/sliders-h?style=solid) icon instead of the cog?

Reply:
> > hide IGDB icon to save on "screen real estate"
> 
> Unfortunately, this is a requirement for being an IGDB partner, we can tweak its appearance but they do want me to display that credit as much as possible, and I'm happy to do so, after all this tool wouldn't exist without IGDB.

I'm all for giving credit, but it makes the searchbar half the screen size. that's a lot of wasted space. Could we move it below the results then?

Reply:
> > make lists as wide as the phone's screen
> 
> I used to have it this way when game board had only 1 list, I ran into issues with the masonry view and instead of fine tuning it I decided to revert the change until I have the time to make it 100% perfect. I would say go for it, again it would be nice to have a separate smaller PR for that change.
> 
> Actually, this is already in place, except that it only applies when list view is not masonry (unique class is applied)

And since I set all my lists to masonry in the past I never noticed that. I'll see what I can do for that view.

Reply:
> > bigger list names
> 
> This one can be very subjective, I've played with the idea of allowing font customization but it would make it very hard to develop as you would have to test on every theme. Maybe we can increase it on mobile only?
> 
> > turn icons bigger
> 
> Open to it as long as it doesn't cause UI issues, I was thinking about maybe making the whole list header clickable instead of having an ellipsis/cog icon.

I'd like that, yeah. On mobile it just needs a few adjustments to make it look like a native app (as a PWA for example).

Reply:
Regarding the request for smaller PRs: don't worry. I just want to clarify things here, since I added a lot of my ideas in this theme, and since you (understandably) don't want major UI adjustments in a single theme I figured we just use this to decide on what can be greenlit in which way.

Reply:
> That's weird, but if the user engagement is up, then it must have a reason. The cog just looks like a global settings icon to me, not a list settings icon. Maybe we could try a sliders icon instead of the cog?

That's based on my small research, don't take me too seriously, there's only like 2.5k users

Reply:
> ... have it 100% when there's only 1 list, I like to limit that behavior to when there's a single list because in mobile it's harder to drag items if the next list is not partially visible.

My idea was to turn each list into something that basically acts (or looks) like its own view on mobile.

You’re on the “currently playing” view, swipe to the side and you’re on the “backlog” view.

Most of the available mobile apps for game tracking work like that, except that they limit you to very few, preset lists. Gamebrary gives the user the choice to have as many lists for as many purposes as they want. That’s liberating.

These apps have a sidebar/navigation bar on the top or bottom with quick access buttons to the 2-3 lists they allow you to manage (backlog, archive, wishlist).

That’s not something Gamebrary could achieve / should attempt as it would create UX problems (too many icons on the navigation bar would look overwhelming or squished and turning it into a slider/carousel just feels 2009-ish).

But to still give the user the look and feel that each list presents itself as the main focus to the user each list would need to fill the whole screen.

What’s your thought about this idea:

Whenever the user drags/long presses a gamecard the game board with all the lists would zoom out a bit (`scale(.8)` or something) and the user would start seeing parts of the lists immediately adjacent to the current one.

Kinda like how you can double press the iPhone’s home button and see the active apps next to each other (except less 3D like).